src/components/ScoreView.vue

```diff
@@ -0,0 +1,124 @@
+<script setup lang="ts">
+import { watch } from 'vue';
+import { useStateStore } from '@/stores/state'
+import Vex from 'vexflow';
+
+
+const state = useStateStore()
+const { Factory, BarlineType } = Vex.Flow;
+
+function isTrebleClef(chord: string | undefined){
+    if(!chord) return false
+
+    const number = Number(chord.charAt(chord.length-1))
```

Comment:
This fails for octaves larger than 9. Also `parseInt(str, 10)` is slightly more stable than `Number`.